examples/stocks/lib/main.dart

```diff
@@ -104,17 +104,19 @@ class StocksAppState extends State<StocksApp> {
       debugPaintSizeEnabled = _configuration.debugShowSizes;
       return true;
     });
-    return new MaterialApp(
-      title: 'Stocks',
-      theme: theme,
-      debugShowMaterialGrid: _configuration.debugShowGrid,
-      showPerformanceOverlay: _configuration.showPerformanceOverlay,
-      routes: <String, RouteBuilder>{
-         '/':         (RouteArguments args) => new StockHome(_stocks, _symbols, _configuration, configurationUpdater),
-         '/settings': (RouteArguments args) => new StockSettings(_configuration, configurationUpdater)
-      },
-      onGenerateRoute: _getRoute,
-      onLocaleChanged: _onLocaleChanged
+    return new SemanticsDebugger(
```

Comment:
before checkin I will remove the SemanticsDebuggers around this app and the earlier one. Later I'll add a way to enable it from MaterialApp.